### PR TITLE
Move cts.logging to _cts.logging.

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: python/pacemaker/_cts/logging.py [hunk 1]
```diff
@@ -1,23 +1,29 @@
-""" Logging classes for Pacemaker's Cluster Test Suite (CTS)
-"""
+""" Logging classes for Pacemaker's Cluster Test Suite (CTS) """
 
-__copyright__ = "Copyright 2014-2020 the Pacemaker project contributors"
-__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+__all__ = ["LogFactory"]
+__copyright__ = "Copyright 2014-2023 the Pacemaker project contributors"
+__license__ = "LGPLv2.1+"
```
Comment (kgaillot) [inline python/pacemaker/_cts/logging.py:5]: This was GPL. Also I prefer the full license line for source code files

File: python/pacemaker/_cts/patterns.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 __all__ = ["PatternSelector"]
 __copyright__ = "Copyright 2008-2023 the Pacemaker project contributors"
-__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+__license__ = "LGPLv2.1+"
```
Comment (kgaillot) [inline python/pacemaker/_cts/patterns.py:5]: ditto, this was fine as it was

**Review by kgaillot — COMMENTED** [review]
looks good

File: python/pacemaker/_cts/logging.py (hunk 1 above)
Comment (clumens) [reply python/pacemaker/_cts/logging.py:5]: I thought we had previously decided that everything in the python directory should be LGPL.

Comment (kgaillot) [reply python/pacemaker/_cts/logging.py:5]: Public stuff definitely, but internal stuff doesn't matter, and we can't unilaterally relicense existing code -- we'd have to contact everyone in the git history for permission

**Review by kgaillot — APPROVED** [review]

**Pull request merged**